### PR TITLE
Logic Fixes and general Coorrections

### PR DESCRIPTION
**Pull request by Vicno**
Added general fixes and comments

### CODE REVIEW

**Comment by vanebustillos** [comment]
Looks good!

File: CampaignModule/BusinessLogic/CampaignLogic.cs [hunk 1]
```diff
@@ -10,58 +10,62 @@ namespace CampaignModule.BusinessLogic
...
         }
 
-        public List<Campaign> Get()
+        public List<Campaign> Get() //Read, returns a list of all its members
         {
             return allCampaign;
         }
 
-        private void Activate()
+        private void Activate() //Deactivates any active campaign present, it considers only one active at the time
```
Comment (bramiprudencio) [inline CampaignModule/BusinessLogic/CampaignLogic.cs:28]: we should change the name for deactivate or lock, it's confused

File: CampaignModule/BusinessLogic/CampaignLogic.cs [hunk 2]
```diff
@@ -10,58 +10,62 @@ namespace CampaignModule.BusinessLogic
...
+
+                
                 break;
             }
         }
 
-        public void Post(Campaign campaign)
+        public void Post(Campaign campaign) //Creates a new Campaign
         {
             Campaign c = allCampaign.LastOrDefault();
```
Comment (bramiprudencio) [inline CampaignModule/BusinessLogic/CampaignLogic.cs]: i like more this "campaign.Id = allCampaign.Count() + 1;"

File: CampaignModule/Database/CampaignTableDB.cs [hunk 3]
```diff
@@ -8,16 +8,22 @@ namespace CampaignModule.Database
...
+                new Campaign() { Id = 5,Name = "Black Friday discounts", Type = "BFRIDAY", Description = "Buy all you can", Active = false}
+    };
+    public List<Campaign> GetAll() //Read, returns all campaigns
+        {
+            return DataBase;
+        }
+
+        public void CUD(List<Campaign> campaigns) // C U D, updates the Database with any changes that may have happened, either Creation, Update or Delete of fields
+        {
+            DataBase = campaigns;
```
Comment (bramiprudencio) [inline CampaignModule/Database/CampaignTableDB.cs:26]: i think you should pass only one campaign and then add to Database

Comment (Vicno) [reply CampaignModule/Database/CampaignTableDB.cs:26]: that would be the right thing to do, but it should still work, besides with that i manage any create, update or delete

**Pull request merged**